### ISSUE TITLE
Fatal error after install on easypanel 🐛

### ISSUE THREAD

**Issue by channchetra** [Easypanel-Community/easyshortner#19]
**Describe the bug**
I'm installed easyshortenter by one click template on easypanel after deployment success
When trying to run the application after installation:

> **Warning:** require(/var/www/html/public/../vendor/autoload.php): Failed to open stream: No such file or directory in /var/www/html/public/index.php on line 34
> 
> **Fatal error:** Uncaught Error: Failed opening required '/var/www/html/public/../vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/html/public/index.php:34 Stack trace: #0 {main} thrown in /var/www/html/public/index.php on line 34

**Version**
Easyshortener version: ghcr.io/easypanel-community/easyshortener:1.1.2

**Deployment Platform**
Easypanel 1.20.1

**Comment by channchetra** [comment]
By the way if I'm using manual installation from this repo directly it's working fine on the same panel it seem occur cus of One click Installation template of that panel.

**Comment by Supernova3339** [comment]
> By the way if I'm using manual installation from this repo directly it's working fine on the same panel it seem occur cus of One click Installation template of that panel.

This is due to dependencies updates within 1.1.2, the image from source works 100% fine. I hope to release an update soon to combat this!

**Comment by channchetra** [comment]
Yeah thank you! I'm using the image from source right now it's fine 🙏

**Comment by Supernova3339** [comment]
> Yeah thank you! I'm using the image from source right now it's fine 🙏

Awesome! Thank you for using Easyshortener ❤️

**Comment by tiagomatosweb** [comment]
I've tried both using on-click installation and manual installation. Both haven't worked so far.
